src/main/java/io/jenkins/plugins/mcp/server/Endpoint.java

```diff
@@ -230,10 +227,6 @@ protected void init() throws ServletException {
                 .resources(resources)
                 .build();
         PluginServletFilter.addFilter((Filter) (servletRequest, servletResponse, filterChain) -> {
-            boolean continueRequest = validateOriginHeader(servletRequest, servletResponse);
-            if (!continueRequest) {
-                return;
-            }
             if (isSSERequest(servletRequest)) {
                 handleSSE(servletRequest, servletResponse);
```

Comment:
At a glance, this code path newly does not validate?

Reply:
nope I cannot see Origin header validation for SSE (which is dead protocol and will should be removed soon)